viewer/core/src/public/migration/Cognite3DViewer.ts

```diff
@@ -155,6 +169,125 @@ export class Cognite3DViewer {
       new THREE.Vector3()
     )
   };
+  /**
+   * Convert pixel coordinates of the cursor to [-1,1]^2 coordinates.
+   * @param pixelX
+   * @param pixelY
+   */
+  private convertPixelCoordinatesToNormalized = (pixelX: number, pixelY: number) => {
```

Comment:
```suggestion
  private convertPixelCoordinatesToNormalized(pixelX: number, pixelY: number) {
```

Reply:
We use "standard function" and not "lambda functions" (unsure what the naming of this is)

Reply:
I notice that we don't follow this standard in ComboControls.ts - this is because it's been migrated from another repository. We should probably change this class as well though (but I dont think you should bother to do that in this PR)

Reply:
Change everything to normal notation and notated that we want to change ComboControls in another PR